pkgs/tvl/default.nix

```diff
@@ -0,0 +1,9 @@
+{ tvlSrc ? builtins.fetchGit {
+    name = "tvl-depot";
+    url = "https://code.tvl.fyi";
+    rev = "4f6f4e6588ebf26ea30d6e98f1b2e618371fc694";
+    ref = "canon";
```

Comment:
Isn’t this missing a sha?

Reply:
I also thought so, but apparently you can't pin a hash when using `builtins.fetchGit`. The documentation is not 💯 % clear in this regard, but I *think* it assumes that the output is fixed if a `rev` is given or `ref` points to a tag.

If you neither of that is the case, `tarball-ttl` is used like for `fetchurl`/`fetchTarball` without a hash.

Maybe someone else can make more sense of the documentation than me (or I'll have to read the source code I guess): <https://nixos.org/manual/nix/stable/#builtin-fetchGit>

I don't particularly like the state of this which is why'd I prefer `fetchTarball` if it were possible with upstream depot.

Reply:
`fetchGit` with a commit hash prompts the same kind of behavior as `fetchTarball` with a output hash: <https://github.com/NixOS/nix/blob/2.3.10/src/libexpr/primops/fetchGit.cc#L103-L118>